format/testdata/doseq_let.txt

```diff
@@ -0,0 +1,4 @@
+(doseq [a
```

Comment:
Please add a test case for `for` as well. You can just put it in this same file (but give it a different filename).